Dockerfile

```diff
@@ -0,0 +1,92 @@
+FROM ubuntu:focal
+USER root
+WORKDIR /data
+ARG ROOTDATADIR=/data
+ARG RPCUSERNAME=user
+ARG RPCPASSWORD=pass
+ARG DGBVERSION=7.17.2
+ARG ARCH=x86_64
+
+ARG MAINP2P=12024
+ARG MAINRPC=14022
+ARG TESTP2P=12026
+ARG TESTRPC=14023
+
+# You can confirm your timezone by setting the TZ database name field from:
+# https://en.wikipedia.org/wiki/List_of_tz_database_time_zones
+ARG LOCALTIMEZONE=Pacific/Auckland
+
+# Set to 1 for running it in testnet mode
+ARG TESTNET=0
+
+# Do we want any blockchain pruning to take place? Set to 4096 for a 4GB blockchain prune.
+# Alternatively set size=1 to prune with RPC call 'pruneblockchainheight <height>'
+ARG PRUNESIZE=0
```

Comment:
For the purpose of keeping this lightweight, should we default the pruning to 1000 blocks or similar @ChillingSilence or @SmartArray ?

Reply:
Personally I think it's worth supporting the option of pruning but having the full chain by default while it's still only ~20GB as it is at present. If we got to 1TB+ I'd say it's probably worth reviewing etc down the line but where we can encourage users to run a *full* node I feel it's worthwhile 🙂

Reply:
I think that sounds fine.  It's easy enough for developers to adjust this setting when building.